### ISSUE TITLE
Slider Counter - v2 to v3

### ISSUE THREAD

**Issue by omniacode** [alvarotrigo/fullPage.js#3825]
Looking to add a slide counter (as per client request) and found the following fiddle created by @alvarotrigo and forked it to change it slightly to reflect my setup.

[https://jsfiddle.net/8y0qhsbz/](url)

However, the fiddle is using v2 of fullPage.js and using this code with v3 of fullPage.js there seems to be issues with the afterSlideLoad callback.

`console.log(totalItems);`

That variable only fires after the first slide has been scrolled, not when the scrolling has ended when scrolling to the parent "section" and throws an error:

`Uncaught ReferenceError: loadedSlide is not defined`

Same goes for any variable defined within that function.

Does anyone have a working example of a Slide Counter or know why the afterSlideLoad callback is behaving in this way with version 3?

Also wondering if afterSlideLoad is the correct callback to be using to create the slide counter in v3 anyway?

**Comment by omniacode** [comment]
So, I even copied this into my project just to see if the "afterSlideLoaded" callback was working and I can't even get an "alert" to show when reaching the specified Section:

```
afterSlideLoaded: function(section, origin, destination, direction) {
    var loadedSlide = this;

    if(section.anchor == 'thirdSection' && destination.index == 1) {
      alert("The first Slide is Ready to Go");
    }
}
```

So I am not sure if it is because my "slides" are not direct descendants of the Section they are in that is causing this to break. I am at a loss.

**Comment by alvarotrigo** [comment]
The callback  is wrong, see the docs:

```js
	afterSlideLoad: function(section, origin, destination, direction){},
```

**Comment by omniacode** [comment]
Why has this been closed, my original question was never answered? While I edited my 2nd response to fix the typo, it is still an issue/question.

The callback is not firing until the first "slide" slides out of view and the 2nd slide shows up - which means the counter doesn't show up until that happens - which is not the desired behavior.

Even if I copy the code 1:1 from the fiddle (which is using Version 2 of the plugin), it doesn't work which means something was changed from v2 to v3.

1. It seems that the callback for "afterSlideLoad" is not the callback to be used for what I am trying to accomplish here, is that correct?

If I have this:

```
afterSlideLoad: function(section, origin, destination, direction) {
  var loadedSlide = this;

  if(section.index == 2 && origin.index == 0) {
    alert("The 3rd section and 1st slide are loaded");
    console.log(origin.index);
  }
}

```
When I get to the section and scroll the first slide, then the alert happens, but then the console is saying that the "index" for the 2nd slide is 0? That doesn't seem right at all? 

2. Do I have to combine different callbacks now to get this to work via fullPage or do I do this separately because it does seem like it is an issue with the slides not being direct children of the fullpage wrapper?

Any insight or push in the right direction would be helpful.
